skll/experiments.py

```diff
@@ -387,13 +394,33 @@ def _parse_config_file(config_path):
     id_col = config.get("Input", "id_col")
     ids_to_floats = config.getboolean("Input", "ids_to_floats")
 
-    # get the cv folds file and make a dictionary from it
+    # get the cv folds file and make a dictionary from it, if it exists
     cv_folds_file = config.get("Input", "cv_folds_file")
+    num_cv_folds = config.get("Input", "num_cv_folds")
     if cv_folds_file:
         cv_folds = _load_cv_folds(cv_folds_file,
                                   ids_to_floats=ids_to_floats)
     else:
-        cv_folds = 10
+        # set the number of folds for cross-validation
+        if num_cv_folds:
+            try:
+                cv_folds = int(num_cv_folds)
+            except:
```

Comment:
Here's the code I see:

```python
        if num_cv_folds:
            try:
                cv_folds = int(num_cv_folds)
            except:
                raise ValueError("The value for cv_folds should be an integer. " +
                                 "You specified {}".format(num_cv_folds))
```

Here's what I am saying it should look like:

```python
        if num_cv_folds:
            try:
                cv_folds = int(num_cv_folds)
            except ValueError:
                raise ValueError("The value for cv_folds should be an integer. " +
                                 "You specified {}".format(num_cv_folds))
```

Note the difference in the `except` statement.